muikku-rest/src/main/java/fi/otavanopisto/muikku/rest/user/UserRESTService.java

```diff
@@ -209,6 +210,18 @@ public Response searchStudents(
       }
     } 
     
+    if (Boolean.TRUE.equals(includeHidden)) {
+      if (!sessionController.hasEnvironmentPermission(MuikkuPermissions.LIST_HIDDEN_STUDENTS)) {
+        if (userEntityId == null) {
+          return Response.status(Status.FORBIDDEN).build();
```

Comment:
Some kind of message in entity() would save a lot of debugging time in the future